src/config/react-query.config.ts

```diff
@@ -23,7 +23,11 @@ export const QUERY_KEYS = {
     accommodations: 'accommodations',
     accomodationReviews: 'accomodation_reviews',
 
+    media: 'media',
+=======
```

Comment:
remove unused indents and characters, please. It left after conflict fix, I guess